.github/workflows/qiita-plugin-ci.yml

```diff
@@ -0,0 +1,156 @@
+# name: Qiita Plugin CI
```

Comment:
Should we work on putting together a "qiita plugin" action? That way the same image could be imported from all plugins.

Reply:
Sure.